pages/osx/tart.md

```diff
@@ -0,0 +1,36 @@
+# tart
+
+> Build, run and manage macOS and Linux virtual machines (VMs) on Apple Silicon.
+> More information: <https://github.com/cirruslabs/tart>.
+
+- Pull a remote VM image:
+
+`tart pull {{acme.io/org/name:tag}}`
+
+- Clone a VM from a local or remote image source:
+
+`tart clone {{source-vm}} {{vm-name}}`
+
+- Create a new Mac VM from an ipsw file:
+
+`tart create --from-ipsw={{[latest|path/to/file.ipsw]}} {{vm-name}}`
```

Comment:
```suggestion
- Create a new Mac VM from a specific ipsw file:

`tart create --from-ipsw={{latest|path/to/file.ipsw}} {{vm-name}}`
```

Almost the same issue as above.
Mandatory fix: `[` and `]` must be removed as not specified syntax in style guide.